src/server.py

```diff
@@ -1,8 +1,301 @@
+import re
+from datetime import datetime
+from typing import Union
+
 from fastapi import FastAPI
 
+from .data_fetcher import DataFetcher
```

Comment:
```suggestion
from data_fetcher import DataFetcher
```

The `.` shouldn't be necessary

Reply:
It threw an ImportError without the `.` when I was running it with `fastapi run src/server.py` like the README says

Reply:
Yeah does the same thing for me. Should be fine to leave it as is.